onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -251,6 +251,10 @@ def bind_cpu_input(self, name, arr_on_cpu):
         :param name: input name
         :param arr_on_cpu: input values as a python array on CPU
         '''
+        # Hold a reference to the numpy object as the bound OrtValue is backed 
```

Comment:
renamed sesssion.py -> onnxruntime_inference_collection.py to better represent that it contains multiple inference related classes